packages/x-components/static-docs/experience-search-and-discovery/empathize.md

```diff
@@ -24,10 +22,10 @@ effortless search experience.
 Being the face of your predictive layer, Empathize comes into play before the search process starts
 and only makes sense when used with other components.
 
-Thanks to the nature of X&nbsp;Components, you can mix and match the components you use within
+Thanks to the nature of Interface&nbsp;X&nbsp;Components, you can [mix and match the components](/explore-empathy-platform/experience-search-and-discovery/readme.md#mix-and-match-your-experience)  you use within
 Empathize. For example, provide shoppers with search ideas and use Empathize to display popular
-search terms and trending products. Or anticipate your shoppers’ intent with recent searches and
-query suggestions. There are lots of handy possibilities that you might want to place close to the
+search terms and trending products. Anticipate your shoppers’ intent with recent searches and
+query suggestions. Or give shoppers access to their [complete search history](my-history.md) and let them control tracking search data. There are lots of handy possibilities that you might want to place close to the
```

Comment:
```suggestion
query suggestions. Or give them access to their [complete search history](my-history.md) so they can control tracking search data. There are lots of handy possibilities that you might want to place close to the
```

Reply:
Are you giving shoppers access to their complete search history just to control tracking? Is giving access to the history a consequence of wanting them to have control over the search data? If so, you're right. Otherwise, if both actions are considered as goals of the functionality, then it's better the first (current) option, where the two actions are listed.